databricks/koalas/tests/test_dataframe.py

```diff
@@ -4546,6 +4546,17 @@ def test_explain_hint(self):
         finally:
             sys.stdout = prev
 
+    def test_repartition(self):
+        kdf = ks.DataFrame({"age": [5, 5, 2, 2], "name": ["Bob", "Bob", "Alice", "Alice"]})
+
+        new_kdf = kdf.spark.repartition(7)
+        self.assertEqual(new_kdf.to_spark().rdd.getNumPartitions(), 7)
```

Comment:
Shall we also test `self.assert_eq(kdf.sort_index(), new_kdf.sort_index())`?

Reply:
Added!